src/components/RoomDropdown/index.tsx

```diff
@@ -215,17 +229,27 @@ const RoomDropdown = ({ button }: Props) => {
                 {isOpen && (
                     <DropDownListContainer ref={ref}>
                         <DropDownList>
-                            <ListItem
-                                onClick={() => void setMemberModalIsOpen(true)}
-                            >
-                                <AddMemberIcon />
-                                Add member
-                            </ListItem>
-                            <ListItem onClick={() => void setModalIsOpen(true)}>
-                                <EditMembersIcon />
-                                Edit members
-                            </ListItem>
-                            <hr></hr>
+                            {roomPrivacy !== RoomPrivacy.Public ? (
```

Comment:
Instead of `?:` operator this could easily be `condition && ()`. `false` and `null` "render" in the same fashion. It's fine though. It's just a note. No need to change.